examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL2/SDL.h>
+
+int main(int argc, char *argv){
```

Comment:
> Given how SDL needs to highjack main to inject initialization code on most platforms...

GCC produce warning, clang produce error and it seems that all windows tests going to fail